fuse/src/gd-mapping.ts

```diff
@@ -36,13 +36,21 @@ export function handlePauserRemoved(event: PauserRemoved): void { }
 
 export function handleTransfer(event: Transfer): void {
 
-  let statistics = TransactionStatistics.load('txStatistics')
+  let txStatistics = TransactionStatistics.load('txStatistics')
 
+  if (txStatistics == null) {
+    txStatistics = new TransactionStatistics('txStatistics')
+  }
+
+  aggregateTransactionStatisticsFromTransfer(event, txStatistics)
+
+  let statistics = Statistics.load('statistics')
   if (statistics == null) {
-    statistics = new TransactionStatistics('txStatistics')
+    statistics = new Statistics('statistics')
   }
 
-  aggregateTransactionStatisticsFromTransfer(event, statistics)
+  statistics.transactionStatistics = txStatistics.id
```

Comment:
i think this can be done only once when "statistics == null"

Reply:
done